yaas/parsers/host.py

```diff
@@ -0,0 +1,89 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import inspect
+
+from .. import host
+from .. import config
+from .. import utils
+
+def define_subcommand(parent_parser):
+  parser = parent_parser.add_parser(
+      'host',
+      help=inspect.getdoc(host))
+
+  subcommands = [
+      host_ls,
+      host_show,
+      ]
+
+  subparsers = parser.add_subparsers()
+  for subcommand in subcommands:
+      subcommand(subparsers)
+
+def host_ls(parent_parser):
+    def ls(args):
```

Comment:
I think that idea is not very Pythonic. Especially when writing a library, I try not to impose my opinion of what *should* be done in favor of allowing whatever *could* be done. (But also, we must balance what could be done and what should be done *in this PR* haha, and I don't think this is one.) Let's kick this discussion down the road a little.